charts/lagoon-logging/values.yaml

```diff
@@ -233,9 +233,12 @@ excludeNamespaces:
 # This may require tweaking to handle high volumes of logs.
 clusterOutputBuffer:
   flush_thread_count: 32
-  timekey: 1m
-  timekey_wait: 10s
+  timekey: 10s # create chunks every 10 secs, therefore possible to flush every 10sec
+  timekey_wait: 0s # don't wait additional time for the timekey to be filled
   timekey_use_utc: true
+  flush_thread_burst_interval: '0' # don't sleep if there is more data to flush
+  flush_mode: immediate # flush immediate, the forward target is in the same cluster so this should be able to handle it
```

Comment:
I did do quite a bit of testing around this configuration and was able to overload the target fluentd fairly easily. I think we could remove this line if it's going to flush in <10s anyway.

Reply:
As discussed, lets keep this since logs-dispatcher seems to handle it okay.